README.md

```diff
@@ -1,4 +1,6 @@
 #GWT Eclipse Plugin
+
+[![Join the chat at https://gitter.im/gwt-plugins/gwt-eclipse-plugin](https://badges.gitter.im/gwt-plugins/gwt-eclipse-plugin.svg)](https://gitter.im/gwt-plugins/gwt-eclipse-plugin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
When I view the content, it appears to be causing an issue with the first H1. Could you hoist this above the #GWT Eclipse Plugin and try that?